src/site/includes/debug.drupal.liquid

```diff
@@ -1,5 +1,6 @@
 {% if buildtype != 'vagovprod' %}
 <script nonce="**CSP_NONCE**" data-template="includes/debug">
+  window.contentData = {{ debug }};
```

Comment:
Adds JSON data to a global variable so we can access in Cypress tests.

Reply:
@ncksllvn this is the only change we need to look at I think, lgtm but wanted to get your 👀 🙂

Reply:
To save a little on the file size can we change the next line to ->

```js
console.info('Metalsmith template generated from content:', window.contentData);
```

Reply:
Thanks - I'll change this in my 2nd PR!